css/variables.css

```diff
@@ -0,0 +1,47 @@
+html {
+  /** Colors **/
+  --color-red: #ec2227;
+  --color-orange: #f79421;
+  --color-black: #000;
+  --color-grey: #666;
+
+  /** Spacer **/
+  --spacer-2xs: 0.25rem; /* 4px */
+  --spacer-xs: 0.5rem;   /* 8px */
+  --spacer-sm: 1rem;     /* 16px */
+  --spacer-base: 1.5rem; /* 24px */
+  --spacer-lg: 2rem;     /* 32px */
+  --spacer-xl: 2.5rem;   /* 40px */
+  --spacer-2xl: 5rem;    /* 80px */
+  --spacer-3xl: 10rem;   /* 160px */
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
```

Comment:
Styles also have to be applied to the individual tags. Currently h6 is not mapped to css variable for h6 font size

Reply:
Done